app/products/calls/screens/call_screen/call_screen.tsx

```diff
@@ -557,6 +557,13 @@ const CallScreen = ({
             // ignore the error because the call screen has likely already been popped async
             Navigation.pop(componentId).catch(() => null);
         }
+
+        // // If we end a call and return from the Thread screen, the "visible screen" is still the thread.
+        if (NavigationStore.getVisibleScreen() === Screens.THREAD) {
```

Comment:
What if I want to stay in the thread?

Reply:
Hey Elias (thought you were still out, that's why I tagged Daniel). This doesn't impact whether the user stays on regular threads. Only if the user is in the call's thread and you arrived there by selecting the `Call Thread` bottom drawer menu item from the Call screen, only then will the thread be removed from the stack.
If we want to change how this works, I probably make a new ticket since it needs some UX input.

Reply:
No worries, I'm still recovering.

Thing is that I may want to stay in the calls thread reading somethin even if the call ends.

Two options come to mind.
1. Show a csll ended illustration
2. Pop the screen as soon as the user returns to the call screen from the thread

Reply:
I like 2, personally. Made a new ticket for both (and get UX input for 1): https://mattermost.atlassian.net/browse/MM-52971 
This PRs code would still be the same (it doesn't force the call thread popping issue that we'll want to fix). Thanks.